modules/ui_patterns_library/tests/src/FunctionalJavascript/UiPatternsLibraryOverviewTest.php

```diff
@@ -24,13 +31,9 @@ class UiPatternsLibraryOverviewTest extends WebDriverTestBase {
   /**
    * {@inheritdoc}
    */
-  protected function setUp() {
+  protected function setUp(): void {
```

Comment:
Let's remove `: void` as we don't do it anywhere else.

Reply:
There are errors without it. It is needed as parent uses such notation.